src/features/Chat/components/ActiveWindow/MessageField.tsx

```diff
@@ -39,8 +38,6 @@ const MessageField = ({ chat }: Props) => {
     try {
       await sendMessage({ userId, message }).unwrap();
     } catch (error) {
-      console.error('Failed to send message:', error);
-      // TODO: Tässä voitaisiin näyttää virheviesti käyttäjälle.
```

Comment:
Removed this, since we already have notifications. But this local state thingy needs rework. We should create a selector that shows isLoading based on pollingParams